BootloaderCommonPkg/Include/Library/CryptoLib.h

```diff
@@ -27,6 +40,12 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define IPP_HASH_CTX_SIZE  256   //IPP Hash context size
```

Comment:
Is this good for SHA384 too?

Reply:
Max size is take care for SHA256.  HASH_CTX refers to

----------
struct _cpHashCtx_rmf {
   IppCtxId    idCtx;                     /* hash identifier   */
   const cpHashMethod_rmf* pMethod;       /* hash methods      */
   int         msgBuffIdx;                /* buffer index      */
   Ipp8u       msgBuffer[MBS_HASH_MAX];   /* buffer            */
   Ipp64u      msgLenLo;                  /* processed message */
   Ipp64u      msgLenHi;                  /* length (bytes)    */
   cpHash      msgHash;                    /* hash value        */
};

Reply:
Type. Max size is take care for SHA512.